Makefile

```diff
@@ -406,11 +405,17 @@ test:
 test-kube-api:
 	$(MAKE) _run_subsystem_test AUTH_TYPE=local ENABLE_KUBE_API=true FOCUS="$(or ${FOCUS},kube-api)"
 
+# An alias for the test target
+subsystem-test: test
+
+# An alias for the test-kube-api target
+subsystem-test-kube-api: test-kube-api
```

Comment:
we have external usage for the aliases?
because if no, I'd like to just rename and if no then all good (and we should transition with the aliases)

Reply:
or course considering ``deploy-test`` as well

Reply:
>we have external usage for the aliases?

A lot of people may be used to the old names and may be using them in scripts, I don't know. As for `test` and `test-kube-api`, they're both used in `openshift/release`:

```bash
grep "make test" **/*assisted* 2>&1 | grep -v directory

ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-main.yaml:    make test
ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-main.yaml:    make test
ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-release-ocm-2.4.yaml:    make test
ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-release-ocm-2.5.yaml:    make test
ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-release-ocm-2.5.yaml:    make test
ci-operator/config/openshift/assisted-service/openshift-assisted-service-cloud_hotfix_releases.yaml:        make test
ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml:        make test
ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml:        make test
ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml:        make test-kube-api
ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml:        make test-kube-api
ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.4.yaml:        make test
ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.4.yaml:        make test
ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.4.yaml:        make test-kube-api
ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.4.yaml:        make test-kube-api
ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.5.yaml:        make test
ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.5.yaml:        make test
ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.5.yaml:        make test-kube-api
ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.5.yaml:        make test-kube-api
```

Reply:
Ack
Lots of the references there are for different repositories, but it does demonstrate we have external usage
Thanks!